lib/shared/pages/loading_page.dart

```diff
@@ -31,7 +31,7 @@ class LoadingPage extends StatelessWidget {
                             semanticLabel: MaterialLocalizations.of(context).backButtonTooltip,
                           )
                         : Icon(Icons.arrow_back_rounded, semanticLabel: MaterialLocalizations.of(context).backButtonTooltip),
-                    onPressed: () => Navigator.of(context).maybePop(),
+                    onPressed: null,

```

Comment:
Just curious, but why was this changed?

Reply:
I disabled back navigation from the loading page because it can be confusing. As it was, you could close the loading page, but then the "real" page (which was always still loading in the background) would pop in eventually, which can be confusing. In other words, the ability to close the loading page gave the false sense of canceling the navigation when it really didn't do anything.

Of course, if we could cancel the request upon closing the loading page, that would be a better solution. But that would be a lot more work. I thought this would lead to the least confusion as things are now.

Reply:
Ahh okay, that makes sense! Thanks for the clarification